gz_ros2_control/src/gz_system.cpp

```diff
@@ -491,9 +491,11 @@ void GazeboSimSystem::registerSensors(
 }
 
 CallbackReturn
-GazeboSimSystem::on_init(const hardware_interface::HardwareInfo & info)
+GazeboSimSystem::on_init(const hardware_interface::HardwareComponentInterfaceParams & params)
 {
-  if (hardware_interface::SystemInterface::on_init(info) != CallbackReturn::SUCCESS) {
+  if (hardware_interface::SystemInterface::on_init(params.hardware_info) !=
```

Comment:
```suggestion
  if (hardware_interface::SystemInterface::on_init(params) !=
```

@soham2560 can we use this overload already?